src/test/scala/tech/cryptonomic/conseil/util/ConfigUtilTest.scala

```diff
@@ -5,6 +5,37 @@ import com.typesafe.config.ConfigFactory
 
 class ConfigUtilTest extends WordSpec with Matchers {
 
+  "the config.Natural matcher" should {
+
+    "match a valid positive integer string" in {
+      "10" match {
+        case Natural(value) => value shouldBe 10
+        case _ => fail("the matcher didn't correcly identify an integer")
+      }
+    }
+
+    "refuse a zero integer string" in {
```

Comment:
why "0" is not a `Natural`?

Reply:
Because the definition is not unambiguous and I needed a name to say the positive integers (the zero is already handled in a separate case handler for our usage of this method)